app/src/renderer/components/governance/PageProposal.vue

```diff
@@ -137,6 +151,41 @@ export default {
       return num.percentInt(
         this.proposal.tally_result.abstain / this.totalVotes
       )
+    },
+    status() {
+      if (this.proposal.proposal_status === `Passed`)
+        return {
+          message: `This proposal has passed`,
+          color: `green`
+        }
+      if (this.proposal.proposal_status === `Rejected`)
+        return {
+          message: `This proposal has been rejected and voting is closed`,
+          color: `red`
+        }
+      if (this.proposal.proposal_status === `DepositPeriod`)
+        return {
+          message: `Deposits are open for this proposal`,
+          color: `yellow`
+        }
+      if (this.proposal.proposal_status === `VotingPeriod`)
+        return {
+          message: `Voting for this proposal is open`,
+          color: `blue`
+        }
+      else
+        return {
+          message: `There was an error determining the status of this proposal.`,
+          color: `grey`
+        }
+    }
+  },
+  watch: {
```

Comment:
what does this accomplish exactly?

Reply:
ohh that watch, I can delete it (one of my attempts to make it work)

Reply:
deleted